bindings.c

```diff
@@ -4563,14 +4638,14 @@ static int cpuview_proc_stat(const char *cg, const char *cpuset, struct cpuacct_
 		threshold = total_sum / cpu_cnt * max_cpus;
 
 		for (curcpu = 0, i = -1; curcpu < nprocs; curcpu++) {
-			if (i == max_cpus)
```

Comment:
Do you have a reason for moving this block? I don't see `i` used for anything other than counting?

Reply:
To fix issue #287 CPU usage is not correct.  The cause and resolution are explained above.